.gitignore

```diff
@@ -8,6 +8,7 @@
 pyvenv.cfg
 /_build
 /styles/Microsoft
+.python-version
```

Comment:
This should go in the user's gitignore, with whatever implementation they choose.

This `.gitignore` should include only those things that are generated by the project, not things that the user may or may not use.

Reply:
https://github.com/plone/buildout.coredev/blob/6.0/.gitignore

Reply:
https://github.com/plone/buildout.coredev/blob/6.0/.gitignore

Reply:
https://github.com/plone/volto/blob/master/.gitignore

Reply:
That's fine for those repos, but not this one.